tools/csv-merger/csv-merger.go

```diff
@@ -657,6 +657,8 @@ The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on O
 	ocsCSV.Annotations["external.features.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["ceph-rbd"], "cephClusters": ["monitoring-endpoint"]}`
 	// Injecting the RHCS exporter script present in Rook CSV
 	ocsCSV.Annotations["external.features.ocs.openshift.io/export-script"] = rookCSV.GetAnnotations()["externalClusterScript"]
+	// Used by UI to track platforms that support External Mode
+	ocsCSV.Annotations["external.features.ocs.openshift.io/supported-platforms"] = `["Baremetal", "None", "VSphere", "OpenStack", "oVirt"]`
```

Comment:
Can we not have "None" as a supported platform?

Reply:
Why?

Reply:
UPI/baremetal env generally reports the infra as None(Non-automated) env. So let's not remove it.

Reply:
If it's not in this list, can we consider it's "None"?

If not, we need at least a comment on the code explaining what "None" is so that it doesn't look like we support 'none' platform.

Reply:
If it's not on the list then it could be Cloud Platforms. I guess we can add a comment explaining what `None` is.

Reply:
Comment for "None" is still needed. Also, `Baremetal` should be `BareMetal`.